examples/helloworld/Makefile

```diff
@@ -1,7 +1,6 @@
-TOOLCHAIN_DIR := ../..
 TARGET = helloworld
 
-C_SOURCES = src/isr.c src/main.c
-ASM_SOURCES = src/startup.asm
+C_SOURCES := src\isr.c src\main.c
+ASM_SOURCES := src\startup.asm
```

Comment:
These backslashes seem wrong

Reply:
They're not. It's due to how some other stuff works (commands like DEL on Windows don't accept / paths). For GNU make on Linux, they're translated in the pm_gnu.mk (mk88 can't translate it from / to \\ so it has to be this way)

Since the tool suite is Windows-centric to begin with, I don't really see this as a problem.